### PR TITLE
[semver:minor] Remove asciidoc build step per upstream

### PR DESCRIPTION
**Pull request by justwheel**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

This commit removes the AsciiDoc configuration logic. The `asciidoctor`
package is proposed to be added to the `cibuilds/hugo` image, which
renders the boolean flag in this Orb redundant.

See https://github.com/cibuilds/hugo/pull/229 for more information.

### Description

Basically undoes what was added in #43.

### CODE REVIEW

**Comment by gmemstr** [comment]
@jwflory While we appreciate the contribution, I would recommend holding off on creating such a PR until the change is actually accepted. Since it's not guaranteed, it's better for there to be a little bit of lag between upstream and an orb rather than having a PR open and waiting for upstream to be reviewed (especially given our current backlog). I'll keep the PR open, but it's unlikely I'll remember to check the upstream PR regularly.

**Comment by justwheel** [comment]
Hi @gmemstr, thanks for the clarification. The upstream pull request is now merged, so this is now ready for review.

**Comment by gmemstr** [comment]
@jwflory I think there's still a reference to the parameter somewhere - happy to merge this in once that's resolved though :)

**Comment by justwheel** [comment]
Oops! Fixed in commit be51a061f29f6a40c7e6e8d17006d54adf22e06c.

**Comment by justwheel** [comment]
Anything to help this move forward? I noticed the failing integration test but it appears to be unrelated to this change, with the Markdown interpreter?

**Comment by justwheel** [comment]
Hi @felicianotech and @KyleTryon, anything else I can do to help this move forward?

**Comment by FelicianoTech** [comment]
Hey,

Bot of your PRs have been merged. We haven't released yet due to failing integration tests. I've fixed them in my PR [here](https://github.com/CircleCI-Public/hugo-orb/pull/50). Once that's merged, your changes will be available in a release.

**Comment by justwheel** [comment]
Thanks, @felicianotech! 🙏🏻 I'll keep an eye on #50. Really appreciate these getting merged in. 🚀
